.envrc

```diff
@@ -1,4 +1,4 @@
-export DATA_DIR="$PWD/data"
+export DATA_DIR="/Volumes/GoogleDrive/Team Drives/GOV.UK teams/2018-2019/Q3/Knowledge up Q3/Data science/data/"
```

Comment:
this should be called something else and the orginial DATA_DIR definition kept as is, so that users can work locally or from the google drive

Reply:
`DATA_DIR_REMOTE`?

Reply:
I've been working on this yesterday. Sorry for not pushing earlier. I've got quite a few changes to append to this PR if you can hold off the review til then...